web/app/components/user-tabs.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  newUser: null,  // this is binded to the text box for adding user
+  actions: {
+
+    /** bubble this action up */
+    setTab: function(tabname) {
+      this.sendAction('setTab', tabname);
+    },
+
+    /** Adds the tab **/
+    addTab: function() {
+      this.sendAction('addTab', this.newUser);
+      Ember.$("#user-input").attr("type","hidden");
```

Comment:
Done. I'm now using a boolean variable called showInputBox to hide and show the input box.